business_requirement/views/business_view.xml

```diff
@@ -0,0 +1,228 @@
+            <field name="name">business.requirement.form</field>
+            <field name="model">business.requirement</field>
+            <field name="type">form</field>
+            <field name="priority">20</field>
+            <field name="arch" type="xml">
+                <form string="Business Requirement">
+                    <header>
+                        <button name="action_button_confirm" states="draft" string="Confirm" type="object" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <button name="action_button_back_draft" string="Back To Draft" type="object" attrs="{'invisible': [('state', 'in', ('draft','done'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <button name="action_button_approve" states="confirmed" string="Approve" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_stakeholder_approval" states="approved" string="Stakeholder Approval" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_in_progress" states="stakeholder_approval" string="In Progress" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_done" states="in_progress" string="Done" type="object" groups="business_requirement.group_business_requirement_manager" class="oe_highlight"/>
+                        <button name="action_button_cancel" string="Cancel" type="object" attrs="{'invisible': [('state', 'in', ('done','cancel'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <button name="action_button_drop" string="Drop" type="object" attrs="{'invisible': [('state', 'in', ('done','drop','cancel'))]}" groups="business_requirement.group_business_requirement_user" class="oe_highlight"/>
+                        <field name="state" widget="statusbar" statusbar_visible="draft,confirmed,approved,stakeholder_approval,in_progress,done,cancel"/>
+                    </header>
+                    <sheet>
+                        <div class="oe_right oe_button_box" name="buttons" groups="base.group_user">
+                             <button class="oe_inline oe_stat_button" type="action" name="%(act_view_sub_br_all)d" icon="fa-tasks">
+                                <field name="sub_br_count" string="Sub Bus. Req." widget="statinfo"/>
+                            </button>                          
+                        </div>
+                        <h1>
+                            <field name="name" class="oe_inline" readonly="1"/>-
+                            <field name="description" class="oe_inline"/>
```

Comment:
Do you think there's a way to make the description field larger on the screen? I find that typically the requirement description is rather larger than what the standard input size has. Not a big deal, but would be nice.